app/app.pro

```diff
@@ -78,7 +78,7 @@ macx {
         hunspell/spellchecker_macx.cpp
 }
 
-unix {
+unix:!macx {
```

Comment:
I done the same in https://github.com/cloose/CuteMarkEd/pull/302, https://github.com/cloose/CuteMarkEd/pull/302/commits/e353102f37c65756966e195df27da8dfb3592edd